### DISCUSSION TITLE
How to draw "24 hours" of tracking on map

### DISCUSSION THREAD

**Discussion by zanac** [Thomas55555/husqvarna_automower#556] [General]
I have setup the map integration and it work... see this screenshot:
![image](https://github.com/Thomas55555/husqvarna_automower/assets/21194919/1e2efdbe-044a-4ee7-8a1b-33f02aa0b401)

Does it exist a way to draw 24 hour of mowing like on "map" card?
![image](https://github.com/Thomas55555/husqvarna_automower/assets/21194919/a7ad3559-ce9a-417e-9de9-c19fc4a893dc)

**Comment by Thomas55555** [comment]
No.
The map card uses the history integration for historical data.
The image entity just the mower data

**Reply by zanac** [reply]
It can be cool to implement image entity to track the historical data with a parameter for choose how hour to show...
